interfaces/cython/cantera/test/test_kinetics.py

```diff
@@ -480,6 +480,16 @@ def test_sticking_coeff_err(self):
                 ct.Interface("sticking_coeff_check.yaml", "Pt_surf", [gas])
 
 
+def check_raises(yaml, err_msg, line=None):
```

Comment:
It seems like `line` is required, any reason to give it a default value?

Reply:
This does indeed work. I was mostly just trying to keep the `line=` when calling for clarity.